test/Driver/imported_modules_bridging_header.swift

```diff
@@ -1,2 +1,7 @@
+// RUN: not %target-swiftc_driver -typecheck %s -import-objc-header %S/Inputs/imported_modules/bridging_header.h -I %S/Inputs/imported_modules 2>&1 | %FileCheck %s
```

Comment:
Ah, instead of %target-swiftc_driver, please use %target-swift-build. That includes a bunch of standard options.